contrib/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @ovn-org/ovn-kubernetes-members
```

Comment:
also don't we have a cap on the github org membership settings? will we get bottlenecked when new people join/ we want to add more nvidia members there as well? (sorry for the silly questions I guess I should check who all are in this list first)

Reply:
we talked about this on slack. The branch protections are in place on master so that people cannot write to master. We can add additional branch protections to stop members from creating other branches in the repo.